core-standards.php

```diff
@@ -2,7 +2,7 @@
 
 /*
   Plugin Name: Core Standards
-  Version: 3.0.3
+  Version: 3.1
```

Comment:
The industry standard https://semver.org/ requires exactly three version identifiers.  I understand you picked up these kind of version numbers from WordPress, but it's an anti-pattern and just wrong.
```suggestion
  Version: 3.1.0
```